lang/en.json

```diff
@@ -487,6 +487,7 @@
         "WarningFileIntegrityNoMd5file": "File integrity check could not be completed due to missing md5_file() function.",
         "WarningPasswordStored": "%1$sWarning:%2$s This password will be stored in the config file visible to everybody who can access it.",
         "WarningDebugOnDemandEnabled": "Tracker %1$s mode is enabled. For security reasons this should be enabled only for a short time frame. To disable it set %2$s to %3$s in %4$s",
+        "WarningDevelopmentModeOnButNotGitInstalled": "You are currently using Matomo in development mode, but it was not installed through git. It's not recommended to use Matomo in development mode in production environment.",
```

Comment:
Not sure if we need this one translatable. Should be a very uncommon case when this is displayed and might be fine to always show the english text.  @tsteur what do you think about that?
If we decide to keep it translatable, we should also add the exception message and the command comment.

Reply:
I just added this into the language file, because it shows on the UI. 
I checked other exceptions and messages in the commands and they are mostly English (at least what I've seen). Kind of tried to follow the actual convention.
But easy to modify to each way, happy to do that.

Reply:
@tsteur what your thoughts on this?

Reply:
Have no thoughts on this. Both is fine. :) And as both is fine I'd just leave it there with a translation as it's less work to simply keep it compared to removing it but really no preference